### PR TITLE
Import from Vim fat finger

### PR DESCRIPTION
**Pull request by Gelma**
Stolen from here:
https://github.com/chip/vim-fat-finger

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7626,6 +7640,7 @@ doign->doing
 doiing->doing
 doiuble->double
 doiubled->doubled
+doller->dollars
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]:
```
doller->dollar
dollers->dollars
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7626,6 +7640,7 @@ doign->doing
...
 dollor->dollar
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: add `dollors->dollars` please

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -12733,9 +12776,11 @@ lible->libel, liable,
 libraies->libraries
 librairies->libraries
 libraris->libraries
+librarry->library
 libray->library
 libreoffie->libreoffice
 libreoficekit->libreofficekit
+librery->library
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: add plural

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -12733,9 +12776,11 @@ lible->libel, liable,
 libraies->libraries
 librairies->libraries
 libraris->libraries
+librarry->library
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: add plural

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -16748,6 +16804,7 @@ pseuo-palette->pseudo-palette
 psitoin->position
 psitoined->positioned
 psitoins->positions
+psot->post
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: add plural

**Comment by larsoner** [comment]
Also needs a rebase

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: > Also needs a rebase

Done

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -17526,6 +17588,7 @@ quesitonable->questionable
 quesitons->questions
 questionaire->questionnaire
 questionnair->questionnaire
+questoin->question
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: add plural

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**